pandas/core/frame.py

```diff
@@ -2295,6 +2295,138 @@ def _from_arrays(
         )
         return cls(mgr)
 
+    @doc(storage_options=generic._shared_docs["storage_options"])
+    def to_rdata(
+        self,
+        path_or_buffer: FilePathOrBuffer,
+        file_format: str = "infer",
+        rda_name: str = "pandas_dataframe",
+        index: bool = True,
+        compression: CompressionOptions = "gzip",
+        storage_options: StorageOptions = None,
+    ) -> None:
+        """
+        Render one or more DataFrames to R data (.RData, .rda, .rds).
+
+        .. versionadded:: 1.3.0
+
+        Parameters
+        ----------
+        path_or_buffer : a valid str, path object or file-like object
+            Any valid string path is acceptable.
+
+        file_format : {{'infer', 'rda', 'rdata', 'rds'}}, default 'infer'
+            R serialization type generated from native commands: base::save
+            (that saves multiple objects) or base::saveRDS (that saves a
+            single object to disk). Default 'infer' will use extension in file
+            name to determine the format type.
+
+        rda_name : str, default "pandas_dataframe"
+            Name for R data.frame in RData/rda file.
+
+        index : bool, default True
+            Include index or MulitIndex in output as separate columns. Since
+            DataFrame indexes can include multiple columns and R rownames can
+            only include one column, DataFrame index will not map to R
+            data.frame rownames.
+
+        compression : {{'gzip', 'bz2', 'xz', None}}, default 'gzip'
+            Compression type for on-the-fly decompression of on-disk data.
+
+        {storage_options}
+
+        Returns
```

Comment:
I think you can drop the Returns section here.  You coudl add a Raises section though.

Reply:
Agreed. Will adjust.